fab/fabfile.py

```diff
@@ -143,8 +143,8 @@ def tag_commit(self):
             sha=self.deploy_ref,
             obj_type='commit',
             tagger={
-                'name': user.name,
-                'email': user.email,
+                'name': user.login,
```

Comment:
was `user.name` null too? If not it might be better to use that or possibly `user.name or user.login` here.